packages-ts/gauntlet-terra-contracts/src/commands/contracts/migration/index.ts

```diff
@@ -0,0 +1,81 @@
+import { Result } from '@chainlink/gauntlet-core'
+import { prompt } from '@chainlink/gauntlet-core/dist/utils'
+import { TerraCommand, TransactionResponse } from '@chainlink/gauntlet-terra'
+import { AccAddress, MsgMigrateContract } from '@terra-money/terra.js'
+import { CONTRACT_LIST } from '../../../lib/contracts'
+
+type CommandInput<MigrationMessage> = {
```

Comment:
From `@terra-money/terra.js` we get message types abbreviated as `Msg`, so let's use the same in our own types to be consistent.

```suggestion
type CommandInput<MsgMigrate> = {
```